airflow/config_templates/config.yml

```diff
@@ -1351,6 +1351,13 @@
       example: "/path/to/my_html_content_template_file"
       default: ~
       see_also: ":doc:`Email Configuration </howto/email-config>`"
+    - name: from_email
+      description: |
+        Email address that will be used as sender address.
+      version_added: 2.2.0
+      type: string
+      example: "airflow@example.com"
+      default: ~
```

Comment:
We should mention this can be either just the “raw” email address, or the “complete” email address format, to avoid users getting confused. And the example should probably use something like `Airflow <airflow@example.com>` as well.